### ISSUE TITLE
Tabbing into inserter doesn't work

### ISSUE THREAD

**Issue by draganescu** [WordPress/gutenberg#22664]
**Describe the bug**
When trying to insert a block from the inline inserter one cannot reach blocks listed by normally using the keyboard.

**To reproduce**
0. Use Firefox
1. Add a post
2. Using the keyboard only:
2.1 Add a columns block
2.2 Tab to the inserter
2.3 Try to tab to select blocks from the inserter menu - fail
2.4 Search for a block, say image
2.5 Try to tab to the result - fail

**Expected behavior**
Same behavior as in chrome where from the search field of the inserter, with or without a search one can tab to the list of blocks and choose what they want to insert.

**Editor version (please complete the following information):**
- WordPress version: [e.g: 5.4]
- "gutenberg plugin
- 8.1.0

**Desktop (please complete the following information):**
 - OS: OSX
 - Browser Firefox
 - Version latest

**Comment by enriquesanchez** [comment]
Hey @draganescu!

I can confirm this is happening on FF.

![2020-05-28 11-11-42 2020-05-28 11_14_02](https://user-images.githubusercontent.com/3276087/83166361-6e2a5380-a0d4-11ea-9dce-f632ef1aa8d9.gif)

Looks like focus leaves the search field for one tab stop, but then returns to the search field.

**Comment by youknowriad** [comment]
I just tested this on Firefox and it seems fixed for me.
